lambda/custom/index.js

```diff
@@ -146,6 +147,24 @@ const main = {
       return response;  
     }  
   },
+    FallbackHandler: {

```

Comment:
there doesn't seem to be a difference between the FallbackHandler and the HelpIntentHandler: why not use the same handler to handle both intents?